.storybook/utils/resetGlobals.js

```diff
@@ -80,4 +79,13 @@ export const resetGlobals = () => {
 			googlesitekit_publish_posts: true,
 		},
 	};
+	global._googlesitekitTrackingData = {
+		referenceSiteURL: 'http://example.com/',
+		userIDHash: 'storybook',
+		activeModules: [],
+		trackingEnabled: false,
+		trackingID: 'UA-000000000-1',
+		userRoles: [ 'administrator' ],
+		isAuthenticated: false,
```

Comment:
We have to remove `trackingEnabled` and `trackingID` from the `global._googlesitekitBaseData = {}` object since we are adding them here.